### PR TITLE
Add boundary figure

### PR DESCRIPTION
**Pull request by scottdtaylor95**

### CODE REVIEW

**Review by aarmey — CHANGES_REQUESTED** [review]
(1) Looks like the build is failing. (2) Can you post what it looks like on here?

You can make it figure S3 or some such so that it can be added to the build.

**Comment by scottdtaylor95** [comment]
Hopefully fixed, it should look like this:
Screenshot (324)

**Review by aarmey — CHANGES_REQUESTED** [review]
Still doesn't build. This should use `getSetup` to setup the figure like all the others do. The plots need to be labelled. The subject points need to be colored by class.

**Comment by aarmey** [comment]
Oh I see... each plot is a separate class. Since class 1 is just the inverse of class 0, the classes can all go on one plot (with coloring to indicate which points are which).

File: syserol/figures/figureS3.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    tFac, _, _ = perform_CMTF()
+    X = tFac.factors[0]
+
+    outt = class_predictions(X)
+    df = load_file("meta-subjects")
+    y = (df["class.nv"] == "viremic").astype(int)
+    y_pred = outt[2][0]
+
+    kern = ConstantKernel() * RBF(np.ones(2), (1e-2, 1e14))
+    kern += WhiteKernel(noise_level_bounds=(0.001, 0.8))
```
Comment (aarmey) [inline syserol/figures/figureS3.py]: This stuff should only be defined in one place. You could make a new function that returns `estG`, then use it here and in `RegressionHelper`?

**Comment by aarmey** [comment]
Ok, reorganized this code a bit to be more organized. Should just need some figure polishing now (labels, titles, tweaks, etc.).

**Pull request merged**